tests/e2e/cucumber/steps/api.ts

```diff
@@ -139,3 +139,17 @@ Given(
     }
   }
 )
+
+Given(
+  '{string} creates the following project space(s) using API',
+  async function (this: World, stepUser: string, stepTable: DataTable): Promise<void> {
+    const user = this.usersEnvironment.getUser({ key: stepUser })
+    for (const space of stepTable.hashes()) {
+      const spaceId = await api.graph.createSpace({ user, space })
+      this.spacesEnvironment.createSpace({
```

Comment:
what is the difference between this two calls? Why do we have to call some `createSpace` twice?

Reply:
the above one which returns `spaceId` it the creation of space with API
And the below one is the creation or storing the space information locally so that it can be get in other required steps without making any explicit request.